### ISSUE TITLE
[Adjust Rule] SIM113: Take care of nested loops

### ISSUE THREAD

**Issue by MartinThoma** [MartinThoma/flake8-simplify#39]
## Desired change

* **Rule(s)**: SIM113
* **Adjustment**: Don't trigger SIM113 in nested loops if the inner loop is counting

## Example

```python
# Code which should NOT trigger SIM113
nb_points = 0
for line in lines:
    for point in line:
        nb_points += 1
```
